src/com/palmergames/bukkit/towny/db/TownyFlatFileSource.java

```diff
@@ -2134,8 +2171,16 @@ public boolean saveTownBlock(TownBlock townBlock) {
 			groupID.append(townBlock.getPlotObjectGroup().getID());
 		}
 		
-		list.add("groupID=" + groupID.toString());
+		list.add("groupID=" + groupID);
```

Comment:
is groupID.toString the same as groupID?

Reply:
Yes